pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go

```diff
@@ -226,3 +387,41 @@ func deserializePipelineRunResources(meta *metav1.ObjectMeta, spec *PipelineRunS
 	}
 	return nil
 }
+
+func convertTaskRunsToChildReference(ctx context.Context, taskRuns map[string]*PipelineRunTaskRunStatus) []v1.ChildStatusReference {
+	csrs := []v1.ChildStatusReference{}
+	for name, status := range taskRuns {
+		csr := v1.ChildStatusReference{}
+		csr.TypeMeta.APIVersion = "v1beta1"
```

Comment:
can you add a comment about why v1beta1 is the right version to use here?

Reply:
@JeromeJu - I think it makes more sense to add the comment explaining why we need v1beta1 about the line that sets the TypeMeta vs adding it to the function signature